ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/NewPooledTransactionHashesMessageProcessor.java

```diff
@@ -111,26 +108,25 @@ private void processNewPooledTransactionHashesMessage(
           incomingTransactionHashes::size,
           incomingTransactionHashes::toString);
 
-      if (syncState.isInSync(SYNC_TOLERANCE)) {
```

Comment:
instead of removing this check entirely, what about having this class subscribe to Synchronizer in sync events?  the first in-sync event would 'open the floodgates':  something like:
```
    syncState.subscribeInSync(inSync -> {
      if (inSync) {
        acceptingRemoteTransactions.set(inSync);
      };
    }, 100000L);
```

Reply:
That will only work if we can achieve sync prior to any PoW clients peering with us.